### ISSUE TITLE
Simpson's rule 

### ISSUE THREAD

**Issue by manildayma** [jainaman224/Algo_Ds_Notes#1892]

**Comment by urbashi123** [comment]
Hi! I would like to work on this issue. Please can you assign me ?
@jainaman224

**Comment by jainaman224** [comment]
@urbashi123 Please select different issue.

**Comment by jainaman224** [comment]
@manildayma Are you working on it?
Please specify the language.

**Comment by manildayma** [comment]
Yes, I'm working on it got busy in some urgent work. I'll do it in js and c# .

**Comment by manildayma** [comment]
I have requested a PR. Please review the same.

**Comment by simrann20** [comment]
@jainaman224 can i work on this issue and implement it in C and C++?

**Comment by Moddy2024** [comment]
@jainaman224  @asha15 i am from gssoc'20 can i work on this issue in python ?

**Comment by Moddy2024** [comment]
@jainaman224 @asha15 @Vaibhavraj10 @raghav-dalmia @MastersAbh i have done the program and sent a pull request please review it before merging

**Comment by Moddy2024** [comment]
@jainaman224 @asha15 @Vaibhavraj10 @raghav-dalmia @MastersAbh can you please assign this to me as i have already written the code in python and this isn't assigned to anyone

**Comment by asha15** [comment]
@manildayma are you working on this?

**Comment by Moddy2024** [comment]
I want to work on this in Python
